cot/tests/router.rs

```diff
@@ -1,23 +1,21 @@
 use bytes::Bytes;
 use cot::config::ProjectConfig;
+use cot::html::Html;
 use cot::project::RegisterAppsContext;
 use cot::request::{Request, RequestExt};
-use cot::response::{Response, ResponseExt};
+use cot::response::{IntoResponse, Response, ResponseExt};
 use cot::router::{Route, Router};
 use cot::test::Client;
 use cot::{App, AppBuilder, Body, Project, StatusCode};
 
-async fn index(_request: Request) -> cot::Result<Response> {
-    Ok(Response::new_html(
-        StatusCode::OK,
-        Body::fixed("Hello world!"),
-    ))
+async fn index(_request: Request) -> Html {
```

Comment:
```suggestion
async fn index() -> Html {
```

This isn't really related to this PR, but we can just drop the `_request` param.